metashare/api/serializers.py

```diff
@@ -83,6 +83,38 @@ def get_repo_url(self, obj) -> Optional[str]:
         return f"https://github.com/{obj.repo_owner}/{obj.repo_name}"
 
 
+class AuthorCommitSerializer(serializers.Serializer):
+    name = serializers.CharField(required=False)
+    email = serializers.CharField(required=False)
```

Comment:
I would think we want to standardize how we're storing commits, whether from `refresh_commits` or a webhook. This has `author` and `name`, but `refresh_commits` has `avatar_url` and `login`.